rfcs/text/0000-data-processing-robustness.md

```diff
@@ -0,0 +1,75 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+DCP data processing robustness principles
+
+## Summary
+
+This document describes requirements for individual DCP components related to robust against incorrect, corrupt, or unexpected data.  It also lays out expectations for data integrity and error handling for DCP software.  It does not specify implementation details or technology.
```

Comment:
_related to robust against_ does not parse. Did you mean _robustness_ ? The authors may find [The Harmful Consequences of the Robustness Principle](https://tools.ietf.org/html/draft-iab-protocol-maintenance-03) to be of interest.

Reply:
IETF draft added as prior work